yt_dlp/extractor/tnaflix.py

```diff
@@ -71,6 +76,10 @@ def get_child(elem, names):
     def _real_extract(self, url):
         mobj = self._match_valid_url(url)
         video_id = mobj.group('id')
+
+        def extract_field(pattern, name):
+            return self._html_search_regex(pattern, webpage, name, default=None) if pattern else None
```

Comment:
This seems unnecessary. But I'll let it be since it's legacy code

Reply:
Decided to use same way to extract data as it's right now in this file